draft-ietf-tls-subcerts.md

```diff
@@ -435,7 +435,7 @@ CertificateEntry of its end-entity certificate; the client SHOULD ignore
 delegated credentials sent as extensions to any other certificate.
 
 The algorithm field MUST be of a type advertised by the client in the
-"signature_algorithms" extension of the ClientHello message and
+"signature_algorithms" extension of the ClientHello message, and
 the dc_cert_verify_algorithm field MUST be of a
 type advertised by the client in the SignatureSchemeList and is
 considered not valid otherwise.  Clients that receive non-valid delegated
```

Comment:
```suggestion
"signature_algorithms" extension of the ClientHello message, and
the dc_cert_verify_algorithm field MUST be of a
type advertised by the client in the SignatureSchemeList; otherwise, the credential is
considered not valid.  Clients that receive non-valid delegated
```